Formula/kube-aws.rb

```diff
@@ -0,0 +1,38 @@
+class KubeAws < Formula
+  desc "Kubernetes on AWS"
+  homepage "https://github.com/coreos/coreos-kubernetes/tree/master/multi-node/aws"
+  url "https://github.com/coreos/coreos-kubernetes/archive/v0.8.1.tar.gz"
+  sha256 "fe65c3fb72af7aa47dad07b8428eace077314edd3c129dba1fea5d3cccfaf094"
+  head "https://github.com/coreos/coreos-kubernetes.git"
+
+  depends_on "go" => :build
+
+  def install
+    executable="bin/kube-aws"
```

Comment:
Could you add spaces on each side of operators? It makes things a lot more readable.